components/brave_ads/browser/ads_service_impl.cc

```diff
@@ -658,6 +662,10 @@ void AdsServiceImpl::OnInitialize(
 
   is_initialized_ = true;
 
+  if (!brave::IsDevOrCanaryBuild()) {
+    SetAdsServiceForNotificationHandler();
```

Comment:
@yachtcaptain23 Ads service should only be set for native ad notifications, i.e. non nightly builds. Can you please code review the `!brave::IsDevOrCanaryBuild()` guard I have added

Reply:
LG2M